_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: post
```

Comment:
You did it almost everywhere already, maybe you can make the remaining sections also one sentence per line. Thanks 😉